---
layout: page
title: People
permalink: /people/
order: 3
---

<h3>Endre Sebestyén</h3>

I obtained my PhD in 2012 at the [Eötvös Loránd
University](https://www.elte.hu/en/), in Classical and Molecular Genetics,
working on the bioinformatic analysis of eukaryotic promoters and transcription
factor binding sites in the group of [Endre Barta](https://abc.naik.hu/szervezeti-egysegek/mbk-genomikai-foosztaly-mezogazdasagi-genomikai-es-bioinformatikai-csoport).
After this, I joined the lab of [Eduardo Eyras](http://comprna.upf.edu/) at UPF,
Barcelona, working on the role of alternative splicing in cancer development and
using splicing patterns as predictive and prognostic signatures of cancer. In
2015, I moved to the lab of [Francesco Ferrari](https://www.ifom.eu/en/cancer-research/research-labs/research-lab-ferrari.php)
at IFOM, Milan, where I worked on developing a novel sequencing method to
characterize large-scale chromatin structure, besides analyzing whole-genome
sequencing datasets to identify driver mutations in non-coding enhancer regions.
My main interests are eukaryotic gene regulatory networks and how they are
changing in different pathologic conditions.

[Twitter](https://twitter.com/endre_sebestyen),
[Google Scholar](https://scholar.google.com/citations?hl=en&user=EFq-0McAAAAJ&view_op=list_works&sortby=pubdate),
[doktori.hu](https://doktori.hu/index.php?menuid=192&lang=EN&sz_ID=30562)

<h3>Péter Szikora</h3>

I am a master student in Medical Biotechnology at [Pázmány Péter Catholic
University](https://itk.ppke.hu/en), focusing on molecular oncology and genomics.
Before that I completed my professional practice in the National Institute of
Environmental Health, working on the molecular analysis of pathogenic bacteria
from hospital water samples. I am currently working on the analysis of
large-scale transcriptome datasets, splicing patterns and gene regulatory
networks in cancer.

<h3>Noam Makmal</h3>
I am a medical student at [Semmelweis University](http://semmelweis.hu/english/).
I'm working on the analysis of genome, transcriptome and ChIP-seq datasets to
develop novel tools for characterizing variance in chromatin structure, link it
to oncogene or tumor suppressor gene expression and ultimately to cancer
development and clinical characteristics of patients.
